view/frontend/email/payment_link.html

```diff
@@ -0,0 +1,16 @@
+<!--@subject {{trans "Order from %store_name is ready for payment" store_name=$store_name}} @-->
+{{template config_path="design/email/header_template"}}
+
+<table>
+    <tr class="email-intro">
+        <td>
+            <p>
+                {{trans "Your order from %store_name for %amount %currency is ready for payment." store_name=$store_name amount=$order.getGrandTotal() currency=$order.getOrderCurrencyCode()}}
+                {{trans "Please visit our payment partner Dintero here to pay the order. This link is valid for %days day(s)." days=$days}}
```

Comment:
I thought this was supposed to be `expiration_days`, but maybe I'm wrong, I don't know the templating system.